assert/assertions_test.go

```diff
@@ -1915,6 +1923,17 @@ func TestInEpsilonSlice(t *testing.T) {
 		0.04), "{2.2, 2.0} is not element-wise close to {2.1, 2.1} in epsilon=0.04")
 
 	False(t, InEpsilonSlice(mockT, "", nil, 1), "Expected non numeral slices to fail")
+
+	inEpsilonSliceMock := new(inEpsilonSliceMockT)
```

Comment:
Why define a new type instead of using the existing `mockT`?
Instead of checking `inEpsilonSliceMock.msgAndArgs` you could use `mockT.errorString()`.

Reply:
>Why define a new type instead of using the existing mockT?

Didn't know that `mockTestingT` exist) 

Thank you for suggestion.